src/WPF/ArcGISRuntime.WPF.Viewer/MainWindow.xaml.cs

```diff
@@ -424,5 +456,45 @@ private void ResetCategories()
             // Set the expanded categories.
             SetExpandedCategories(expandedCategoryNames);
         }
+
+        private void ScreenshotButton_Click(object sender, RoutedEventArgs e)
+        {
+            GetJpgImage(SampleContainer);
+        }
+
+        private void GetJpgImage(UIElement source)
```

Comment:
We should link to the relevant stack overflows in a comment here.
https://stackoverflow.com/q/24466482
https://stackoverflow.com/a/15537372